src/grafana/deploy/grafana-configuration/prom-datasource.json.template

```diff
@@ -1,7 +1,7 @@
 {
   "name":"PM",
   "type":"prometheus",
-  "url":"{{ cluster_cfg['prometheus']['url'] }}",
+  "url":"{{ cluster_cfg['prometheus']['url'] }}/prometheus",
```

Comment:
This is not due to pylon, it is because if we want prometheus be able to accept request from `pylon_url/prometheus`, then prometheus must have `--web.route-prefix=prometheus` set, and grafana also need to request `prometheus_url/prometheus`, same applies to alert-manager.